container_images/daisy-builder/main.sh

```diff
@@ -21,13 +21,13 @@ WORKFLOW_FILE="$3"      # Workflow to run
 GCS_OUTPUT_BUCKET="$4"  # Destination for artifacts
 
 function generate_new_version() {
-  local VERSION=`/versiongenerator --token-file-path=${GITHUB_ACCESS_TOKEN} --org=${REPO_OWNER} --repo=${REPO_NAME}`
+  local VERSION_OUT=`/versiongenerator --token-file-path=${GITHUB_ACCESS_TOKEN} --org=${REPO_OWNER} --repo=${REPO_NAME}`
```

Comment:
does versiongenerator print errors to stdout or stderr?

Reply:
yes, it does write errors for stdout